contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerEntrypoint.sol

```diff
@@ -89,13 +89,23 @@ contract OVM_SequencerEntrypoint {
             s
         );
 
-        Lib_SafeExecutionManagerWrapper.safeCALL(
+        (bool success, bytes memory returndata) = Lib_SafeExecutionManagerWrapper.safeCALL(
             gasleft(),
             target,
             callbytes
         );
+
+        if (success) {
+            assembly {
+                return(add(returndata, 0x20), mload(returndata))
```

Comment:
Why do we need assembly here? 
Is it possible that the length of `returndata` is less than 32 bytes?

Reply:
Can't use `return` in a fallback function without assembly.

Reply:
I see. 
> Is it possible that the length of returndata is less than 32 bytes?
Whoops, I was thinking of storage encoding here.